docs/excel/configure-your-add-in-to-use-the-browser-runtime.md

```diff
@@ -0,0 +1,114 @@
+---
+ms.date: 02/06/2020
+title: "Configure your Excel add-in to share the browser runtime (preview)"
+ms.prod: excel
+description: Configure your Excel add-in to share the browser runtime and run ribbon, task pane, and custom function code in the same runtime.
+localization_priority: Priority
+---
+
+# Configure your Excel add-in to share the browser runtime (preview)
+
+[!include[Running custom functions in browser runtime note](../includes/excel-shared-runtime-preview-note.md)]
+
+When running Excel on Windows or Mac, your add-in will run code for ribbon buttons, custom functions, and the task pane in separate JavaScript runtime environments. You can configure your Excel add-in to share code in a single browser runtime. This enables better coordination across your add-in for sharing code and data. It also enables you to run code when the document opens, or to run code while the task pane is closed. To configure your add-in to share the browser runtime, follow the instructions in this article.
+
+## Create the add-in project
+
+If you are starting a new project, follow these steps to use the Yeoman generator to create an Excel add-in project. Run the following command and then answer the prompts with the following answers:
+
+```command&nbsp;line
+yo office
+```
+
+- Choose a project type: **Excel Custom Functions Add-in project**
+- Choose a script type: **JavaScript**
+- What do you want to name your add-in? **My Office Add-in**
+
+![Screenshot of answering prompts from yo office to create the add-in project.](../images/yo-office-excel-project.png)
+
+After you complete the wizard, the generator creates the project and installs supporting Node components.
+
+## Configure the manifest
+
+Follow these steps for a new or existing project to configure it to use a shared context browser runtime.
+
+1. Start Visual Studio Code and open the **My Office Add-in** project.
+2. Open the **manifest.xml** file.
+3. Change the `<Requirements>` section to use **SharedRuntime** version **1.1** as shown in the following code.
```

Comment:
Please check with Shaofeng about this, but I think this step of adding SharedRuntime as a Requirement is optional. 

- The add-in could use a shared runtime _purely_ for purposes of sharing global variables between the CF and the task pane and not use any of the APIs in the SharedRuntime req set at all. 
- The add-in could use APIs in the SharedRuntime req set, but it could have an alternate experience that doesn't use these APIs, in which case the dev wants the add-in to be installable even on Office platforms that don't support the SharedRuntime req set. 

In either of these scenarios, the dev should not add this markup to the Requirements section because doing so prevents the add-in from being installed on Office platforms that don't support the SharedRuntime req set.